sql/core/src/test/resources/test-data/bad_after_good.csv

```diff
@@ -1,2 +1,2 @@
 "good record",1999-08-01
-"bad record",1999-088-01
+"bad record",1999-088_01
```

Comment:
I had to change this because `FastDateFormat` is not so strong, and can parse `1999-088-01`

Reply:
do we run these tests with legacy formatter?

Reply:
Yes, I added `CSVLegacyTimeParserSuite` which runs entire `CSVSuite` with the legacy parser.